src/main/java/anodot/stage/destination/AnodotTarget.java

```diff
@@ -103,6 +105,12 @@ public void destroy() {
     public void write(Batch batch) throws StageException {
         try {
             Iterator<Record> records = batch.getRecords();
+            Record lastRecord = getLastRecord(batch);
```

Comment:
`records` could be passed instead of `batch `

Reply:
write method is a part of the interface and it accepts batch

Reply:
yes, but it doesn't make much difference